docs/source/conf.py

```diff
@@ -70,13 +70,6 @@
 # The full version, including alpha/beta/rc tags.
 release = fsspec.__version__
 
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
```

Comment:
This now produces a warning when it previously didn't. Removing it gives exactly the same behaviour, i.e. it uses the default language which is `en`.